epimodel/exports/epidemics_org.py

```diff
@@ -382,23 +422,52 @@ def process_export(args) -> None:
     traces_v3_df: pd.DataFrame = pd.read_csv(traces_v3, index_col="CodeISO3")
 
     hopkins_df: pd.DataFrame = pd.read_csv(
-        hopkins, index_col=["Code", "Date"], parse_dates=["Date"]
+        hopkins,
+        index_col=["Code", "Date"],
+        parse_dates=["Date"],
+        keep_default_na=False,
     ).pipe(aggregate_countries, args.config["state_to_country"])
     foretold_df: pd.DataFrame = pd.read_csv(
-        foretold, index_col=["Code", "Date"], parse_dates=["Date"]
+        foretold,
+        index_col=["Code", "Date"],
+        parse_dates=["Date"],
+        keep_default_na=False,
     )
 
     analyze_data_consistency(
-        args.debug, export_regions, models_df, rates_df, hopkins_df, foretold_df
+        args.debug,
+        export_regions,
+        models_df,
+        initial_df,
+        rates_df,
+        hopkins_df,
+        foretold_df,
     )
 
     for code in export_regions:
-        reg = args.rds[code]
+        reg: Region = args.rds[code]
         m49 = int(reg["M49Code"])
         iso3 = reg["CountryCodeISOa3"]
+
+        ##### TODO: go over simulation groups (mitigations), for each get stats and plots
+        for group in set(simulation_specs.Group):
+            sim_ids = list(simulation_specs[simulation_specs.Group == group].index)
+            stats = batch.generate_sim_stats(cummulative_active_df.xs(key=reg.Code, level="Code"), sim_ids)
+            # TODO: do something with the stats
+            # print(reg, group, stats)
+            # TODO: plot Active as a curve
+            # TODO: Add interpolated traces between seasonalities
+
+        # TODO clean this up
+        initial_estimate = int(
+            estimates_df[estimates_df.index.isin(reg.AllNames)]["Final"]
```

Comment:
@mathijshenquet this is where I now got stuck because US is missing here.

Reply:
I have no idea why US is missing. I did not put it back because it is not in the spreadsheat either (it is in a different tab though).

Reply:
See also here: https://epidemicforecasting.slack.com/archives/CV93A58G7/p1587044266125600

Reply:
I temporarily disabled `US` in config.yaml

Reply:
I have no idea why US is missing, it is also missing in the spreadsheet, it is in another tab though. See the discussion here: https://epidemicforecasting.slack.com/archives/CV93A58G7/p1587044266125600. I did not feel confident enough to put it back from the other tab.

Reply:
OK :+1: